pkg/config/threescale.go

```diff
@@ -123,6 +122,31 @@ func (t *ThreeScale) GetReplicasConfig(inst *integreatlyv1alpha1.RHMI) map[strin
 		setDefaultNumberOfReplicas(2, threeScaleComponents)
 	}
 
+	if inst.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		switch inst.Status.Quota {
+		case "1":
+			threeScaleComponents["apicastProd"] = 1
+			threeScaleComponents["backendListener"] = 1
+			threeScaleComponents["backendWorker"] = 1
```

Comment:
I am not sure if these values are ok, I guess I need to get definitive answer if 1M SKU has to be HA. But these can be changed later easily. I would go with all "3"s here though.

Reply:
Hi @trepel I believe there's an ask to confirm the 1 million sku. Once that's confirmed it needs updating in the config map and here. As I understand it that ask is with you. Can you let me know if it's not on your backlog so that we can make sure someone has it.

Reply:
Yeah, I am working on that, but I don't play with replica counts, just with requests/limits for containers. We use 3 as minimum due to HA (in multiAZ setup there are three zones so we can have one pod per zone). But yes, for now we can use whatever here, I just think '3's are more probable the right values than '1's

Reply:
Probably yes but we need to have a quota which is low enough to run in prow. @philbrookes proposed a 0 sku for that. so we can update the 1 to 3 if required.